### ISSUE TITLE
Conflict with local Postgres port

### ISSUE THREAD

**Issue by amercader** [boxkite/datacats#50]
On running `datacats create`:

```
datacats create spatial22 
Creating environment "spatial22"............
Installing ckan requirements
Installing ckan
Installing ckanext-spatial22theme
Initializing database
Traceback (most recent call last):
  File "/home/adria/dev/pyenvs/datacats/bin/datacats", line 9, in <module>
    load_entry_point('datacats==0.7', 'console_scripts', 'datacats')()
  File "/home/adria/dev/pyenvs/datacats/src/datacats/datacats/cli/main.py", line 116, in main
    return command_fn(opts)
  File "/home/adria/dev/pyenvs/datacats/src/datacats/datacats/cli/create.py", line 40, in create
    ckan_version=opts['--ckan'],
  File "/home/adria/dev/pyenvs/datacats/src/datacats/datacats/cli/create.py", line 76, in create_environment
    return finish_init(project, start_web, create_sysadmin)
  File "/home/adria/dev/pyenvs/datacats/src/datacats/datacats/cli/create.py", line 135, in finish_init
    project.start_web()
  File "/home/adria/dev/pyenvs/datacats/src/datacats/datacats/project.py", line 456, in start_web
    self._run_web_container(port, command)
  File "/home/adria/dev/pyenvs/datacats/src/datacats/datacats/project.py", line 522, in _run_web_container
    5000: port if is_boot2docker() else ('127.0.0.1', port)},
  File "/home/adria/dev/pyenvs/datacats/src/datacats/datacats/docker.py", line 139, in run_container
    port_bindings=port_bindings)
  File "/home/adria/dev/pyenvs/datacats/local/lib/python2.7/site-packages/docker/client.py", line 904, in start
    self._raise_for_status(res)
  File "/home/adria/dev/pyenvs/datacats/local/lib/python2.7/site-packages/docker/client.py", line 119, in _raise_for_status
    raise errors.APIError(e, response, explanation=explanation)
docker.errors.APIError: 500 Server Error: Internal Server Error ("Cannot start container 0a4fb176342eac40017b7af8b2926e085eac2546649aa1a82eed38ab55e54bad: Error starting userland proxy: listen tcp 127.0.0.1:5432: bind: address already in use")
```

Doing `datacats purge` and `datacats init` returned the same output.

**Comment by wardi** [comment]
drat. docker-py changed the exception they raise, now I just get an APIError instead of a PortAllocatedError when containers fail to start due the port being in-use.

**Comment by wardi** [comment]
scratch that. The error text just changed.  fixed by #56
